sbt-ash/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperPlugin.scala

```diff
@@ -0,0 +1,65 @@
+package org.virtuslab.ash
+
+import sbt.Def
+import sbt.Keys._
+import sbt._
+
+object AkkaSerializationHelperPlugin extends AutoPlugin {
+
+  object autoImport extends AkkaSerializationHelperKeys {
+    val baseAshSettings: Seq[Def.Setting[_]] = AkkaSerializationHelperPlugin.baseAshSettings
```

Comment:
Hmmm so it's `ashSettings` or `baseAshSettings` that can be overwritten by the end users? 🤔

Reply:
Yes indeed

Reply:
Erm... which one? :trollface:

Reply:
Neither, you override keys, not `Seq[Def.Setting[_]]`

Reply:
Okay... so why the split into `ashSettings` and `baseAshSettings`? 🤔